spec/models/content_migration/course_copy_assignments_spec.rb

```diff
@@ -153,6 +153,12 @@
       assignment_model(:course => @copy_from, :points_possible => 40, :submission_types => 'file_upload', :grading_type => 'points')
       @assignment.turnitin_enabled = true
       @assignment.vericite_enabled = true
+      @assignment.vericite_settings = {
```

Comment:
We probably should change this into two specs.  As a whole we treat turnitin and vericite enabled to be mutually exclusive.  That said, since they share a common backend this is probably okay, but it does go against our general behavior.

That said, this code probably has a lifetime of about a year, so I'm inclined to be okay with it.